README.md

```diff
@@ -25,3 +25,14 @@ at your option.
 Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in the work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any
 additional terms or conditions.
+
+##### BROKEN ON WINDOWS MSVC
```

Comment:
This isn't exactly true, everything works fine on MSVC as long as you put your OpenCL.lib in the right spots (I need to check the details, it's been a long time since I've played with this on Windows). Perhaps something like 'MSVC Caveats' might be a better title.